data-prepper-api/src/main/java/org/opensearch/dataprepper/model/event/EventMetadata.java

```diff
@@ -35,4 +36,26 @@ public interface EventMetadata extends Serializable {
      * @since 1.2
      */
     Map<String, Object> getAttributes();
+
```

Comment:
Do we want to support removing tags as well? Do we need that for this initial implementation or is that something we can add later?

Reply:
I think initial implementation is fine as is. When we add the `tag_manager` processor we will need to add support for removing

Reply:
+1, I was just not sure of the requirements for this feature in the near term.